java/test/jmri/jmrix/openlcb/OlcbClockControlTest.java

```diff
@@ -58,6 +58,10 @@ public void propertyChange(PropertyChangeEvent propertyChangeEvent) {
         }
     }
 
+    @BeforeAll
+    static public void checkSeparate() {
+        org.junit.Assume.assumeFalse("Ignoring intermittent test", Boolean.getBoolean("jmri.skipTestsRequiringSeparateRunning"));
```

Comment:
I'm fine with this PR, but I think it would be good to add a comment on why these tests are on separate tests. Usually that's because a test is randomly failing, but these are not doing that (at least if I understand it correctly). And therefore, someone else might get the idea to move them back to the ordinary test suite since they almost never fail. So a comment like below might be useful for these tests:

`jmri.jmrix.openlcb tests leave 138 threads active. Therefore these tests are moved to separate running. See Issue #10490`

Reply:
Good idea!  Thanks.